webui/src/pages/repositories/index.jsx

```diff
@@ -35,15 +34,15 @@ const LOCAL_BLOCKSTORE_SAMPLE_REPO_DEFAULT_BRANCH = "main";
 
 const CreateRepositoryButton = ({variant = "success", enabled = false, onClick}) => {
     return (
-        <Button variant={variant} disabled={!enabled} onClick={onClick}>
-            <RepoIcon/> Create Repository
+        <Button variant={variant} disabled={!enabled} onClick={onClick} className="d-flex align-items-center">
+            <RepoIcon className="me-2"/> Create Repository
         </Button>
     );
 }
 
-const GettingStartedCreateRepoButton = ({text, variant = "success", enabled = false, onClick, creatingRepo, style = {}}) => {
+const GettingStartedCreateRepoButton = ({text, variant = "success", enabled = false, onClick, creatingRepo, className = ""}) => {
```

Comment:
Not sure if it's needed, but it's worth checking MSB if touching this file.

The good thing is that it's covered by the Enterprise E2E,
So updating the OSS reference in the Ent and opening a PR should validate this on the CI 😎